rules/windows/process_creation/proc_creation_win_susp_elevated_system_shell_uncommon_parent.yml

```
@@ -0,0 +1,69 @@
+title: Elevated System Shell Spawned From Potentially Uncommon Parent Location
+id: 178e615d-e666-498b-9630-9ed363038101
+related:
+    - id: 61065c72-5d7d-44ef-bf41-6a36684b545f
+      type: similar
+status: experimental
+description: Detects when a shell program such as the Windows command prompt or PowerShell is launched with system privileges from a potentially uncommon parent location.
```

```suggestion
title: Elevated System Shell Spawned From Uncommon Parent Location
id: 178e615d-e666-498b-9630-9ed363038101
related:
    - id: 61065c72-5d7d-44ef-bf41-6a36684b545f
      type: similar
status: experimental
description: Detects when a shell program such as the Windows command prompt or PowerShell is launched with system privileges from a uncommon parent location.
```
We know it's an uncommon parent location, don't we? (potentially just reads odd in that position)